packages/design-system/vue/cypress/support/ActionBar.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <header class="actionBar">
+    <div class="actionBar-themeSelect">
+      <label for="themeSwitcher">Theme: </label>
+      <select
+        id="themeSwitcher"
+        name="themeSwitcher"
+        @change="changeTheme"
+      >
+        <option value="dark">
+          Dark
+        </option>
+        <option value="light">
+          Light
+        </option>
+      </select>
+    </div>
+  </header>
+  <main>
+    <slot />
+  </main>
+</template>
+
+<script setup lang="ts">
+function changeTheme(event: Event) {
+  const theme = (event.target as HTMLInputElement).value;
+  document.querySelector('html')?.setAttribute('class', theme);
```

Comment:
Out of scope for this PR, but I'm wondering if it would be better to define our variables on something like `.eds-theme-dark` instead of tying it to `:root` so we don't need to "break out" of our app to set the theme class